DOCS-11868 adding content to modify email alerts (#456)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/configure-alerting.adoc

| `etcd` | Sends an error when the etcd master is unaccessible for longer than 5 min.
| `etcd_latency_batch` | Sends a warning when follower-leader latency exceeds 500ms, and triggers an error if it exceeds 1 second over a one-minute period.
| `filesystem` | Sends a warning when 80% of capacity is used. Triggers a critical error when more than 90% is used.
| `high-cpu` | Sends a warning when 75% of capacity used. Triggers a critical error when more than 90% is used.
| `high_memory` | Sends a warning when 80% of memory is used. Triggers a critical error when more than 90% is used.
| `influxdb_health_batch` | Triggers an error if InfluxDB is down or if there is no connection between Kapacitor and InfluxDB.
| `ingress` | Triggers a critical error when more than 80% of capacity is used.
| `kubernetes_node` | Sends a warning when more than 90% of CPU is used. Triggers a critical error when more than 95% is used.
| `muleapps` | Triggers a critical error when more than 90% of CPU is used. 
| `network_params` | Triggers a critical error if networking is disabled on a node. 
| `uptime` | Sends a warning when the system is inaccessible for longer than 5 min.
|===

== Configure Alerts

> You can send Runtime Fabric alerts from a designated alert sender email address through your SMTP server. You can configure only one alerts email recipient.

Reviewer: What about: You can configure only one email recipient for alerts.

Reviewer: one alerts email... sounds wrong but I don't think it is technically.

> You can send Runtime Fabric alerts from a designated alert sender email address through your SMTP server. You can configure only one alerts email recipient.

Author: ```suggestion
You can send Runtime Fabric alerts from a designated alert sender email address through your SMTP server. You can configure only one email recipient for alerts.
```

$ sudo ./rtfctl appliance apply alert-smtp-from "sender@example.com"
----

== Modify Email Alert Contents

If needed, you can modify the content of email alerts sent from Runtime Fabric. Alerts are written in TICKscript.

[NOTE]
--

> If you customize any Runtime Fabric email alerts, appliance upgrades overwrites the customizations. 

Reviewer: ```suggestion
If you customize any Runtime Fabric email alerts, appliance upgrades overwrite the customizations. 
```

--

. Navigate to Ops Center.
. From the Ops Center menu, click *Configuration*.
. From the *Config maps* drop-down menu, select *kapacitor-alerts*.
. From the *Namespace* drop-down menu, select *monitoring*.
. Select the alert you want to modify, and make changes.
. When complete, click *Apply*.
. Use SSH or Ops Center to log in to one of the controller nodes. 
. Get the kapacitor pod name:
+
----
kubectl -n monitoring get pod -l component=kapacitor
----
+
The name will look like `kapacitor-xxxxxxxxxx-xxxx`.
. Get a list of configured alerts:
+
----
kubectl -n monitoring exec <pod name> -c alert-loader -- kapacitor -url http://localhost:9092 list tasks
----
. Delete the alert you modified:
+
----
kubectl -n monitoring exec <POD NAME> -c alert-loader -- kapacitor -url http://localhost:9092 delete tasks <MODIFIED ALERT NAME>
----

> . Check that the alert is recreated and the status/executing values are enabled/true:

Reviewer: ```suggestion
. Verify that the alert is re-created, the Status value is `enabled`, and the Executing value is `true`:
```
